rdkit/Chem/UnitTestSurf.py

```diff
@@ -8,83 +7,87 @@
 #  which is included in the file license.txt, found at the root
 #  of the RDKit source tree.
 #
-""" unit testing code for surface calculations
-
-FIX: add tests for LabuteASA
-
+"""
+unit testing code for calculations in rdkit.Chem.MolSurf
 """
 from __future__ import print_function
-from rdkit import RDConfig
-import unittest, os
-from rdkit.six.moves import cPickle
-from rdkit import Chem
-from rdkit.Chem import MolSurf
+
+from collections import namedtuple
 import os.path
+import unittest
 
+from rdkit import Chem
+from rdkit import RDConfig
+from rdkit.Chem import MolSurf
 
-def feq(n1, n2, tol=1e-4):
-  return abs(n1 - n2) <= tol
+doLong = False
+TestData = namedtuple('TestData', 'lineNo,smiles,mol,expected')
 
 
 class TestCase(unittest.TestCase):
+  dataNCI200 = os.path.join(RDConfig.RDDataDir, 'NCI', 'first_200.tpsa.csv')
+  dataNCI5000 = os.path.join(RDConfig.RDCodeDir, 'Chem', 'test_data', 'NCI_5K_TPSA.csv')
+  dataTPSAregr = os.path.join(RDConfig.RDCodeDir, 'Chem', 'test_data', 'tpsa_regr.csv')
 
-  def setUp(self):
-    if doLong:
-      print('\n%s: ' % self.shortDescription(), end='')
+  @classmethod
+  def readNCI_200(cls):
+    for data in readPSAtestData(cls.dataNCI200):
+      yield data
 
-  def testTPSAShort(self):
-    " Short TPSA test "
-    inName = RDConfig.RDDataDir + '/NCI/first_200.tpsa.csv'
-    with open(inName, 'r') as inF:
-      lines = inF.readlines()
-    for line in lines:
-      if line[0] != '#':
-        line.strip()
-        smi, ans = line.split(',')
-        ans = float(ans)
-
-        mol = Chem.MolFromSmiles(smi)
+  @classmethod
```

Comment:
Makes no difference here, as there will never be more than one instance and the use of the class is for testing only. 

In general, it's an indicator for me that the method doesn't access object specific.